calculator.go

```diff
@@ -1,13 +1,90 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
 // Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+func Add(a float64, b ...float64) float64 {
```

Comment:
We'd like to be able to enforce that _at least two_ arguments are passed, wouldn't we? It doesn't make much sense to call Add (or Subtract, or Multiply) with only one argument.

Reply:
Great point! Fixed.